components/esp_websocket_client/esp_websocket_client.c

```diff
@@ -1204,33 +1208,27 @@ int esp_websocket_client_send_fin(esp_websocket_client_handle_t client, TickType
 
 int esp_websocket_client_send_with_opcode(esp_websocket_client_handle_t client, ws_transport_opcodes_t opcode, const uint8_t *data, int len, TickType_t timeout)
 {
-    int ret = ESP_OK;
+    int ret = -1;
```

Comment:
Isn't this changing the original logic?

Reply:
The logic is unchanged, just the default return value is `-1`, so each time we fail we don't have to re-assign this to `-1`.
(the original logic didn't actually use this value anywhere, as if all the subsequent check passed we set `ret = esp_websocket_client_s...()` )